news/2017-08-17-happystick-summer-tour-concludes.md

```diff
@@ -0,0 +1,50 @@
+After an arduous and exciting five weeks of matches in which more than 700 initial contestants were narrowed down to a mere 26, the HappyStick Summer Tour reached its climax Sunday, August 13 with the Grand Final matches. Over the course of the tournament, 768 players from [x] countries across 13 rank brackets ranging from the 250 - 500 to 100,000+ tiers fought their way through a grand total of 537 matches from Group Stages through Bracket Stages until only two players remained in each. Those select few fought it out mano a mano on the grand stage of the finals where they clashed for the crown of Summer Tour Champion.
+
+![](/wiki/shared/2017-08-17-happystick-summer-tour-concludes/banner.jpg)
```

Comment:
banner max width should be 300px, ideal is 200px, width no more than 1400 tops. i use 1350x200px same as FA pages just out of convenience and it seems to work pretty well
well

always jpg 90-100, 90 preferable unless it fucks up gradients and makes them look like shit

should also be in the folder structure as seen above: ``/wiki/shared/<newspost filename>/something.jpg``